### ISSUE TITLE
Adopting project

### ISSUE THREAD

**Issue by rousveiga** [dgomes/home-assistant-custom-components#3]
Hello, 
I'd like to adopt the USB-to-DALI Home Assistant integration to make it work with newer Home Assistant versions. Would you mind if I forked and maintained it?

**Comment by dgomes** [comment]
Please go ahead :)

By the way, I'm now using https://github.com/dgomes/dali2mqtt

**Comment by rousveiga** [comment]
Thank you! Yes, I saw that, perfect solution if you want a separate DALI controller. However, in my case, I have a USB DALI master which I'd like to connect directly to my RasPi where I'm running Home Assistant. Using MQTT in this situation seems like overkill to me.

**Comment by rousveiga** [comment]
Just to make sure I'm not overlooking anything - you had this running on a venv installation, right? I'm having library issues getting it to work with HASS OS.

**Comment by dgomes** [comment]
Yes it was running on venv environment, tag me on your repository with your issues, maybe I can help

**Comment by rousveiga** [comment]
Thanks, but I think I figured it out! It was a pyhidapi issue; turns out HASS OS is an Alpine Linux and pyhidapi wasn't looking for the hidapi library in the right places. I think I just need to fork pyhidapi and replace a few paths. I'll ask you if I run into further problems, if you don't mind 🙂

**Comment by rousveiga** [comment]
I was wondering if you could help me with a problem I'm having. Now I got Home Assistant to load the integration and it recognizes the DALI master properly; however, it doesn't seem to discover any DALI drivers. In this piece of code:
```python
   for lamp in range(0,config[CONF_MAX_GEARS]):
        try:
            _LOGGER.debug("Searching for Gear on address <{}>".format(lamp))
            r = dali_driver.send(gear.QueryControlGearPresent(Short(lamp)))
            if isinstance(r, YesNoResponse) and r.value:
                lamps.append(Short(lamp))
        except Exception as e:
            #Not present
            _LOGGER.error("Error while QueryControlGearPresent: {}".format(e))
            break

    add_devices([DALILight(dali_driver, lock, config[CONF_NAME], l) for l in lamps])
```
the `if` condition is never true, so new entities never get added. I've already ruled out any physical connection issues. What would you think this has to do with?
Thanks in advance.

**Comment by dgomes** [comment]
OK.. that is one of the reasons why I deprecated this custom_component.

The API of python-dali changed **a lot**... you are better starting from scratch and using dali2mqtt as an example implementation of how to use python-dali

**Comment by rousveiga** [comment]
I see, will do. I'll try to run dali2mqtt and see if that sheds any light. Thank you!
